README.md

```diff
@@ -468,6 +469,72 @@ ANSWERS.addComponent('DirectAnswer', {
 })
 ```
 
+## Custom Direct Answer Card
+
+```js
+  class MyClassName extends ANSWERS.Component {
+    constructor(config = {}, systemConfig = {}) {
+      super(config, systemConfig);
+
+      // Add your Handlebars template here.
+      this.setTemplate(`
+        <div class="myDirectAnswer">
+          My Direct Answer Custom Template
+          <div class="myDirectAnswer-type">
+            {{type}}
+          </div>
+          <div class="myDirectAnswer-value">
+            {{directAnswerPrefix}}
+            {{answer.value}}
+          </div>
+        </div>
+      `)
+    }
+
+    /**
+     * Here, data is the directAnswer data from the query. You can expect an object that looks something
+     * like:
+     * {
+     *   type: "FIELD_VALUE",
+     *   answer: {
+     *     entityName: "Entity Name",
+     *     fieldName: "Phone Number",
+     *     fieldApiName: "mainPhone",
+     *     value: "+11234567890",
+     *     fieldType: "phone" 
+     *   },
+     *   relatedItem: { 
+     *     verticalConfigId: 'people',
+     *     data: { ... }
+     *   }
+     * }
```

Comment:
I think you should separate this out. Maybe start the section by showing what the directAnswer data looks like. Then, have a blurb about how you can base a custom DA card around this object with an example (like the one you have currently).

Reply:
tried to separate things, as well as include similar logic to what is currently in the theme